gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1142,6 +1159,70 @@ reconcile_when_close_toggled_cb (GtkToggleButton *togglebutton, ofx_info* info)
     info->run_reconcile = gtk_toggle_button_get_active (togglebutton);
 }
 
+static void
+runMatcher(ofx_info* info, char * selected_filename, gboolean go_to_next_file)
+{
+    GtkWindow *parent = info->parent;
+    GList* trans_list;
+    GList* trans_list_remain = NULL;
+    Account* first_account = NULL;
+    
+    // If we have multiple accounts in the ofx file, we need to process transactions one account at a time,
+    // in case there are transfers between accounts.
+    trans_list = info->trans_list;
+    info->num_trans_processed = 0;
+    for(;trans_list;trans_list=trans_list->next)
+    {
+        Transaction* trans = trans_list->data;
+        Split* split = xaccTransGetSplit (trans, 0);
+        if (first_account == NULL) first_account = xaccSplitGetAccount (split);
+        if (xaccSplitGetAccount (split) == first_account)
+        {
+            gnc_gen_trans_list_add_trans (info->gnc_ofx_importer_gui, trans_list->data);
+            info->num_trans_processed ++;
+        }
+        else trans_list_remain = g_list_prepend (trans_list_remain, trans_list->data);
+    }
+    g_list_free (info->trans_list);
+    info->trans_list = trans_list_remain;
+    
+    // See whether the view has anything in it and warn the user if not.
+    if (gnc_gen_trans_list_empty (info->gnc_ofx_importer_gui))
+    {
+        gnc_gen_trans_list_delete (info->gnc_ofx_importer_gui);
+        if (info->num_trans_processed)
+        {
+            gchar* acct_name = gnc_get_account_name_for_register (first_account);
+            gnc_info_dialog (parent, _("OFX file '%s', imported transactions for account '%s'\n%d transactions processed, no transactions to match"),
```

Comment:
As [some of our weblate translators found](https://github.com/Gnucash/gnucash/pull/1144/files), this has broken thee string freeze.

Reply:
@jeanlaroche
While translating myself, I found it hard to understand. Have a look at the right column of
https://hosted.weblate.org/translate/gnucash/gnucash/en/?&offset=3677#history
for some of the problems.

Perhaps it looks nicer and is easier to understand with a colon after the first and a full stop after the second line.

Can you submit a fix after the string freeze is lifted?

Reply:
Sorry about that, I'm not 100% sure what I should have done to avoid breaking things. 
In any case, can you clarify a bit what you think would be clearer? Can you give me an example of the format you think would be easier to understand?

Reply:
> Sorry about that, I'm not 100% sure what I should have done to avoid breaking things.

You wrote it before the string freeze and on merging @jralls didn't see the changed string.
Usually we make no msgmerge in in te frozen time, but a translator found an unmarkes string.
So it was a concatenation of rare events.

Reply:
Ah, I start to understand. 
If it is a message box, use natural language. Only for table cells use short terms.

How about:
OFX import from file %s into account %s: \n …

How it the relation between  processing and matching?
%d transactions were processed, but no matchs were found.
or what do you try to express?

Reply:
The meaning of this message is
%d transactions were processed, all of which were previously imported

(so you don't need to match any, hence we won't show you the matching dialog) would be the very-long version!
Note that I changed the string, but did not remove the _() macro, so I'm not sure how the string change was not detected.

Reply:
The history:
2021-09-10: your commit as PR
2021-09-13 my msgmerge after string freeze (without your change)
2021-09-16: John's  merge of your PR into maint
2021-09-20: my extraordinary msgmerge into the .po files (with your change)
2021-09-21: translators started to update their translation

Just try to formulate as message normal sentences, which the user will understand. In some cases you might want to add some context for us translators—we do not see it in the context of the process and related GUI elements. Then add a [translator comment](https://wiki.gnucash.org/wiki/I18N#Translator_Comments_for_Single_Messages) to tell us the context.